python/huggingfaceserver/huggingfaceserver/generative_model.py

```diff
@@ -218,6 +218,7 @@ def load(self) -> bool:
             tokenizer_kwargs["trust_remote_code"] = True
 
         model_kwargs["torch_dtype"] = self.dtype
+        tokenizer_kwargs["clean_up_tokenization_spaces"] = True
```

Comment:
Does this apply to all model types?

Reply:
@yuzisun Thanks a lot for the comment! I added this because some test failed as `clean_up_tokenization_spaces` is false by default in transformers v4.45. I thought in most cases, we do not expect a whitespace before punctuations, but you are right, transformers might update this for freedom to clean up or not.  So I think I should leave it as `False` by default, and just set this to be True during decoding.